build.psm1

```diff
@@ -1012,7 +1012,7 @@ function Start-PSPester {
     # All concatenated commands/arguments are suffixed with the delimiter (space)
```

Comment:
@JamesWTruher, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/test/tools/CodeCoverageAutomation/README.md.https://coveralls.io/github/PowerShell/PowerShell should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/testing-guidelines/TestRoadmap.md.https://coveralls.io/github/OpenCover/opencover should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```